Observer/SalesOrderSaveAfterObserver.php

```diff
@@ -98,6 +98,6 @@ private function isOrderUnholded(Order $order): bool
 
     private function isOrderCancelled(Order $order): bool
     {
-        return ($order->isCanceled()) && $order->getOrigData('state') != 'cancelled';
+        return ($order->isCanceled()) && $order->getOrigData('state') != 'canceled';
```

Comment:
```suggestion
        return ($order->isCanceled()) && $order->getOrigData('state') != Magento\Sales\Model\Order::STATE_CANCELED;
```

what do you think about using the constant instead?

Reply:
Makes sense, thanks!